generated/_task_modules/channels/ai_channel.py

```diff
@@ -549,61 +564,13 @@ def ai_accel_units(self):
             self._handle, self._name)
         check_for_error(error_code)
 
-    @property
-    def ai_acceld_b_ref(self):
```

Comment:
Ugh, the original name was horribly mangled since dB makes camelCasing confused. I'm OK if this changes as long as the new one is correct. I think it should be ai_accel_db_ref. I see it in `ATTRIBUTES_BLACKLIST`, but I forget what that does.

Reply:
BTW, "dB" tends to confuse the camel-to-snake-case conversions, so `ai_acceld_b_ref` may not be the name we really want in the end.

Cc: @zhindes @maxxboehme

Reply:
I think it changed to `ai_sound_pressure_db_ref`. The `_db_ref` part is fine but I'm not sure about the change from "accel" to "sound_pressure".

```
PS C:\dev\nidaqmx-python> rg db_ref
generated\_task_modules\channels\ai_channel.py
7733:    def ai_sound_pressure_db_ref(self):
7757:    @ai_sound_pressure_db_ref.setter
7758:    def ai_sound_pressure_db_ref(self, val):
7771:    @ai_sound_pressure_db_ref.deleter
7772:    def ai_sound_pressure_db_ref(self):
9143:    def ai_voltage_db_ref(self):
9165:    @ai_voltage_db_ref.setter
9166:    def ai_voltage_db_ref(self, val):
9179:    @ai_voltage_db_ref.deleter
9180:    def ai_voltage_db_ref(self):
```

Reply:
It looks like both `accel` and `sound_pressure` should exist: https://www.ni.com/docs/en-US/bundle/ni-daqmx-properties/page/daqmxprop/attr29b2.html and https://www.ni.com/docs/en-US/bundle/ni-daqmx-properties/page/daqmxprop/attr29b1.html. I think this one is getting blacklisted, and sound pressure is getting handled correctly.